test/infrastructure/docker/Dockerfile

```diff
@@ -27,8 +27,8 @@ COPY go.sum go.sum
 # Essentially, change directories into CAPD
 WORKDIR /workspace/test/infrastructure/docker
 # Copy the Go Modules manifests
-COPY test/infrastructure/docker/go.mod go.mod
-COPY test/infrastructure/docker/go.sum go.sum
+COPY test/go.mod go.mod
+COPY test/go.sum go.sum
```

Comment:
I guess we should apply a similar change in the tiltfile, right?

Reply:
I investigated this a little bit and given the code at

https://github.com/kubernetes-sigs/cluster-api/blob/376dca06eb95ff03f4b156772588761b45c2f80d/Tiltfile#L155-L160

With this change we are loosing live reload for go.mod and go.sum

https://github.com/kubernetes-sigs/cluster-api/blob/376dca06eb95ff03f4b156772588761b45c2f80d/Tiltfile#L74-L75

However, I consider this a small price to pay, so I'm ok with it